dask_jobqueue/core.py

```diff
@@ -607,6 +620,73 @@ def _new_worker_name(self, worker_number):
             cluster_name=self._name, worker_number=worker_number
         )
 
+    def _get_worker_security(self, security):
+        """Dump temporary parts of the security object into a shared_temp_directory"""
+        if security is None:
+            return None
+
+        worker_security_dict = security.get_tls_config_for_role("worker")
+
+        # dumping of certificates only needed if multiline in-memory keys are contained
+        if not any(
+            [value is not None and "\n" for value in worker_security_dict.items()]
```

Comment:
Found a last problem: is always true as in value is missing in a copy and paste error!